scdl/scdl.py

```diff
@@ -450,6 +457,12 @@ def download_track(track, playlist_name=None, playlist_file=None):
                 logger.debug(e)
         else:
             logger.error("This type of audio doesn't support tagging...")
+            
+        #Try to change the real creation date
+        created_at = track['created_at']
+        filetime = int(time.mktime(datetime.strptime(created_at, '%Y/%m/%d %H:%M:%S %z').timetuple()))
```

Comment:
You can have a look at https://github.com/flyingrub/scdl/pull/147/commits/0736cb0c1e2f226678e19b14de3ae1e371c385c0#diff-834ccb1f68cfeac6d63b0b550a320ba0R467 for inspiration